config.TEMPLATE.inc.php

```diff
@@ -505,3 +505,17 @@
 ; This setting overrides the 'curl.cainfo' parameter of the php.ini configuration file.
 [curl]
 ; cainfo = ""
+
+
+;;;;;;;;;;;;;;;;;;;
+; Queues Settings ;
+;;;;;;;;;;;;;;;;;;;
+
+[queues]
+
+; Default queue driver to be used
+default_connection = "database"
+; Default queue to be used if someone forgot to associate a queue during Queue::push();
+default_queue = "queue"
```

Comment:
If a user doesn't remember to add this entry to their configuration file, `default_queue` will be `null`. I'd recommend providing a sensible default in calls to `Config::getVar`, as you've done for `default_connection` already. (Also applies to OMP and OPS.)

Reply:
I'm setting the default value at [BaseJob](https://github.com/pkp/pkp-lib/pull/7049/files#diff-91b0f26b34266d092022bceaf10c468644b536a8fa5b90917cf75b83f7029d88R54).

What do you think in adding "queue" as default hardcoded value on the BaseJob?